src/main/java/nl/mpcjanssen/simpletask/ThemedPreferenceActivity.kt

```diff
@@ -6,7 +6,13 @@ import nl.mpcjanssen.simpletask.util.Config
 abstract class ThemedPreferenceActivity : AppCompatPreferenceActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        setTheme(Config.activeTheme)
+        setTheme(Config.activeTheme({ it ->
+            when (it) {
+                "dark" -> R.style.AppTheme_Settings
+                "black" -> R.style.AppTheme_Black_Settings
+                else -> R.style.AppTheme_Light_DarkActionBar_Settings
+            }
+        }))
```

Comment:
This is the reason why I made the change -- to allow each class to set its own styles based on the theme, specifically because the settings don't look good with the selection color from the main theme.